core/dbt/events/core_types_pb2.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
 # source: core_types.proto
-# Protobuf Python Version: 5.26.1
+# Protobuf Python Version: 4.25.3
```

Comment:
Looks like the protobuf version changed from the autogeneration. 5.26.1 is a pain to install but I have it locally, will re-generate and push that up.

Reply:
done 👍